### PR TITLE
feat: add component `CCard`

### PR DESCRIPTION
**Pull request by ItsPierre**
Requires #52

### CODE REVIEW

File: src/components/CCard.vue [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+);
+</script>
+
+<style scoped lang="scss">
+.c-card .c-card-sections > * {
+  &:not(:first-child):not(:last-child),
+  &:first-child:not(:last-child) {
+    margin-bottom: 12px;
+  }
+}
```
Comment (johannes-huther) [inline src/components/CCard.vue]: Just for clarity:
You add 12px of margins for every child of `.v-card-sections`, if:

- It is not the first element and not the last element
- It is the fist element, but not the last element

Correct? If so: Should that work as well?
```suggestion
.c-card .c-card-sections > * {
  &:not(:last-child) {
    margin-bottom: 12px;
  }
}
```

**Review by johannes-huther — CHANGES_REQUESTED** [review]

File: src/components/CCard.vue (hunk 1 above)
Comment (ItsPierre) [reply src/components/CCard.vue]: ah yes, i had this like that because I tried some more conditions earlier

**Comment by ItsPierre** [comment]
https://github.com/Crystal-Creations-GbR/crystal-components/compare/0f4bec36e8a8dd1f42463e8cd3ee8611009d0e83..b70cef126746062f7b5e9432415ac118f59094b3

**Review by johannes-huther — CHANGES_REQUESTED** [review]
Please add story

File: stories/CCard.stories.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  render: createStorybookRender({
+    components: { CCard },
+    template: `
+<c-card v-bind='args' class="ma-5">
+  <template #content-full-width="{ padding }">
+      <v-list>
+          <v-list-item :class="padding" @click="() => {}">This content has the full width</v-list-item>
+          <v-divider></v-divider>
+          <v-list-item :class="padding" @click="() => {}">Use 'padding' prop to set matching padding.</v-list-item>
+      </v-list>
```
Comment (johannes-huther) [inline stories/CCard.stories.ts:112]: I believe we should add a `slotTemplate` prop to `createStorybookRender`, but we can do that later

**Review by johannes-huther — APPROVED** [review]

**Comment by johannes-huther** [comment]
I believe we should add a `slotTemplate` prop to `createStorybookRender`, but we can do that later

**Pull request merged**